src/main/resources/EmailService.properties

```diff
@@ -56,3 +56,16 @@ email.joining.signature=<p>If you have questions, you can reply to this email or
+email.patient.when.crc.changes.htmlBody=<p>Dear %{SalutationFirstName},</p><p>The research coordinator associated with your participation in the <a href="http://${HOSTNAME}">Cancer Moonshot<sup style="display: inline-block; text-decoration: none;">SM</sup> Biobank</a> has changed. Their contact information has been updated in your account settings.</p><p>&nbsp;</p><table border="0" width="100%" cellspacing="0" cellpadding="0"><tbody><tr><td><div><!-- [if mso]><v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" data-route="true" href="https://${HOSTNAME}/dashboard" style="height:36px;v-text-anchor:middle;width:250px;" arcsize="5%" strokecolor="#1976d2" fillcolor="#1976d2"><w:anchorlock/><center style="color:#ffffff;font-family:Helvetica, Arial,sans-serif;font-size:14px;">View your account settings</center></v:roundrect><![endif]--><a style="background-color: #1976d2; border: 1px solid #1976d2; border-radius: 4px; color: #ffffff; display: inline-block; font-family: Helvetica, Arial,sans-serif; font-size: 16px; line-height: 40px; text-align: center; text-decoration: none; width: 300px; -webkit-text-size-adjust: none; mso-hide: all;" href="https://${HOSTNAME}/dashboard" data-route="true">View your account settings</a></div></td></tr></tbody></table><p>&nbsp;</p><p>Button not working? Copy and paste this link in your browser: <a href="http://${HOSTNAME}/dashboard/participant/%{PatientId}">http://${HOSTNAME}/dashboard/participant/%{PatientId}</a></p>
+
+#Common Signature #2
+email.thankyou.signature=<p>Thank you for participating in the Cancer Moonshot\u2120 Biobank!</p><p>- Cancer Moonshot Biobank Staff</p><p>Questions? Reply to this email or call us at 202-000-0000.</p>
```

Comment:
The common signature is 
<p>- Cancer Moonshot Biobank Staff</p><p>Questions? Reply to this email or call us at 202-000-0000.</p>

There are three thak you lines which we'll have to deal with separately